sql/expression/function/convert_tz.go

```diff
@@ -173,6 +184,22 @@ func getDeltaAsDuration(d string) (time.Duration, error) {
 	return time.ParseDuration(symbol + hours + "h" + mins + "m")
 }
 
+func getSystemDelta() string {
+	t := time.Now()
+	_, offset := t.Zone()
+	seconds := offset % (60 * 60 * 24)
+	hours := math.Floor(float64(seconds) / 60 / 60)
+	seconds = offset % (60 * 60)
+	minutes := math.Floor(float64(seconds) / 60)
+
+	result := fmt.Sprintf("%02d:%02d", int(math.Abs(hours)), int(math.Abs(minutes)))
+	if offset >= 0 {
+		result = fmt.Sprintf("+%s", result)
+	}
```

Comment:
Seems like we also need to handle negative timezone offsets. 

It would be helpful to break out a private helper function that does this logic and has the system timezone offset passed in as a parameter – that would allow us to write tests against this specific logic more easily, without having to always rely on what the system `TZ` settings is configured to.

Reply:
Now the offset is coming as an argument to the function.